package.json

```diff
@@ -16,6 +16,7 @@
     "cypress": "6.0.1",
     "ory-prettier-styles": "1.1.1",
     "prettier": "2.2.1",
+    "remarkable": "^2.0.1",
```

Comment:
Shouldn't this be in `docs/package.json`?

Reply:
Yes that was a mistake, have to watch my packages better...